src/backend/joanie/core/enums.py

```diff
@@ -21,30 +21,23 @@
 ORDER_STATE_PENDING = "pending"  # waiting for payment and enrollment
 ORDER_STATE_CANCELED = "canceled"
 ORDER_STATE_FAILED = "failed"
-ORDER_STATE_IN_PROGRESS = "in_progress"
+ORDER_STATE_PAID = "paid"
```

Comment:
ah ok I didn't know what this state was for as it wasn't used. "in progress" is indeed not a good name if it meant that it is cleared for payment. Ok for "validated".

Reply:
After thought. I think "paid" is better than "validated" because the latter sounds like someone approved something. And we might need it for this kind of purposes later...
From my experience with ERPs it is standard to have an order/invoice in a "paid" state even if it is for 0.00€. In the state machine we can send an order of 0.00€ directly to the "paid" state without going through the payment step...

Also don't forget that we won't have any orders at all for free enrollments so this case won't be common.